tests/ruby/fasterer_test_files/module_eval.rb

```diff
@@ -0,0 +1,15 @@
+module Hihi
+  class << self
+    module_eval %{
+      def hello
+        puts "win"
+      end
+    }
+  end
+end
+
+Hihi.hello
+
+Hihi.module_eval %{
+ puts @foo
```

Comment:
two space indent please.

Also pls review https://github.com/coala/coala-bears/pull/2561 so we can merge that and you can rebase on top of it. to avoid conflicting it.

Reply:
Done @jayvdb